aws/resource_aws_cognito_user_pool_test.go

```diff
@@ -883,7 +885,7 @@ resource "aws_cognito_user_pool" "test" {
 
   admin_create_user_config {
     allow_admin_create_user_only = false
-    unused_account_validity_days = 7
+    unused_account_validity_days = 6
```

Comment:
Is there a reason to change this in the test?

Reply:
Once support for new field is added, it is no longer possible to update this field as described [here](https://github.com/terraform-providers/terraform-provider-aws/pull/10890#issuecomment-573464578).

And because we're using `nil` value for this in case of error from AWS API, it is no longer possible to update this value.